### PR TITLE
peer discovery: UI cleanup

### PR DESCRIPTION
**Pull request by micahscopes**
### Summary
Fixes three peer importing UI issues:
- makes the "Searching" spinner fade in and out smoothly
- ensures that the `KModal` fade transition isn't blocked
- prevents the "no devices yet" message from showing abruptly before initial network location fetching has even completed.

### Reviewer guidance
Try the peer importing dialog and make sure everything looks smooth!

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [-] If there are any front-end changes, before/after screenshots are included
- [x] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue [hunk 1]
```diff
@@ -218,6 +218,8 @@
             this.savedAddresses = addresses;
             this.resetSelectedAddress();
             this.stage = this.Stages.FETCHING_SUCCESSFUL;
+            this.savedAddressesInitiallyFetched = true;
+            this.saved = true;
```
Comment (jonboiser) [inline kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue]: What's `this.saved` for? I don't see it in the `data` object

Comment (micahscopes) [reply kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue]: ah, I think that is a fluke mistake!

Comment (micahscopes) [reply kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue]: I've deleted it

**Review by jonboiser — APPROVED** [review]
looks good to me!

**Comment by indirectlylit** [comment]
did this PR address these "dual modal" issues here?

image


* https://www.notion.so/learningequality/0-13-0-bug-bash-3-7eeb57a21dd14aaa8f672dca9981f3f9?p=ab0cbc5da2ed4e25935c54898a29484f
* https://www.notion.so/learningequality/0-13-0-bug-bash-3-7eeb57a21dd14aaa8f672dca9981f3f9?p=c1e5d9c45c04432db891c1b95d1500be

**Comment by micahscopes** [comment]
@indirectlylit yes, the blocking modal has been addressed.  It turned out to be an issue of my using the name "fade" for a transition, which is also used by `KModal`.  @jonboiser recommended that maybe `KModal` could get a better name for its fade transition that isn't likely to be used in components that are inserted into its slots.

**Comment by indirectlylit** [comment]
thank you

**Pull request merged**